### PR TITLE
Feature/enable swap kstore

### PR DESCRIPTION
**Pull request by canercandan**

### CODE REVIEW

File: x/clp/types/errors.go [hunk 1]
```diff
@@ -37,4 +37,5 @@ var (
 	ErrAsymmetricAdd                   = sdkerrors.Register(ModuleName, 35, "Cannot add liquidity asymmetrically")
 	ErrAsymmetricRemove                = sdkerrors.Register(ModuleName, 36, "Cannot remove liquidity asymmetrically")
 	ErrAsymmetricRatioAdd              = sdkerrors.Register(ModuleName, 37, "Cannot add liquidity with asymmetric ratio")
+	ErrSwapInterrupted                 = sdkerrors.Register(ModuleName, 38, "Unable to swap, swap is currently interrupted")
```
Comment (joneskm) [inline x/clp/types/errors.go:40]: Change to "Unable to swap, swaps are currently disabled"?

Comment (joneskm) [inline x/clp/types/errors.go:40]: Change to "Unable to swap, swaps are currently disabled"?

**Review by timlind — APPROVED** [review]
lgtm

**Comment by canercandan** [comment]
Closing this PR in favor of https://github.com/Sifchain/sifnode/pull/2816

**Pull request closed without merging**